### PR TITLE
GEN-2430 - styles(bonnus-offer): added basic recommendation card

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- Added basic recommendation card designs

Obseve tho that "See your price" links are not working due an issue related with 'priceIntentAtoms'.
Gonna be addressing that in the next PR

Screenshot 2024-09-02 at 14 02 47

## Justify why they are needed

Bonus offer.

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
* **#4672** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4672?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @guilhermespopolin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/BonusOfferPresenter.tsx [hunk 1]
```diff
@@ -31,19 +32,11 @@ export function BonusOfferPresenter() {
 
   return (
     <div className={yStack({ gap: 'xl' })}>
-      {/* TODO: Lokalise this */}
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/BonusOfferPresenter.tsx:34]: I guess bundle discount text will come in a later PR?

File: apps/store/src/features/priceCalculator/BonusOfferPresenter.tsx [hunk 2]
```diff
@@ -62,6 +55,15 @@ function RecommendationCard({ recommendation }: { recommendation: Recommendation
         <ButtonNextLink href={recommendation.product.pageLink} variant="secondary" size="medium">
           {t('READ_MORE')}
         </ButtonNextLink>
+        {recommendation.product.priceCalculatorPageLink && (
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/BonusOfferPresenter.tsx:58]: Please check for feature flag too

**Review by alebedev — APPROVED** [review]
Looks good

**Review by alebedev — APPROVED** [review]

File: apps/store/src/features/priceCalculator/BonusOfferPresenter.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/features/priceCalculator/BonusOfferPresenter.tsx:34]: Yes

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Sep 11, 4:44 AM EDT**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4672).

**Pull request merged**